xbmc/video/dialogs/GUIDialogAudioSubtitleSettings.cpp

```diff
@@ -204,7 +204,7 @@ void CGUIDialogAudioSubtitleSettings::OnSettingAction(const CSetting *setting)
       strPath = g_application.CurrentFileItem().GetPath();
 
     std::string strMask = ".utf|.utf8|.utf-8|.sub|.srt|.smi|.rt|.txt|.ssa|.aqt|.jss|.ass|.idx|.rar|.zip";
-    if (g_application.GetCurrentPlayer() == "videoplayer")
+    if (g_application.GetCurrentPlayer() == "VideoPlayer")
```

Comment:
ok, let's take this PR as is, thanks to @wiromare 
we still can do improvements later